modules/pool/withdraw/components/PoolWithdrawSingleAsset.tsx

```diff
@@ -59,6 +64,22 @@ export function PoolWithdrawSingleAsset({ onShowPreview, ...rest }: Props) {
             />
             <PoolWithdrawSummary mt="6" />
             <PoolWithdrawSettings mt="8" />
+            <Collapse in={hasHighPriceImpact} animateOpacity>
+                <Alert status="error" borderRadius="md" mt="4">
+                    <Checkbox
```

Comment:
Please refer to how i changed the layout on the invest alert

Reply:
I'm not sure I understand this comment. The invest and withdraw Alerts are the same. Did you mean swap? you removed the borderRadius and no transition. Is that they way you want it?